src/main/java/se/poochoo/reminder/ReminderHelper.java

```diff
@@ -38,10 +31,46 @@
     static final long UPDATE_NOTIFICATION_INTERVAL = 30000;
     static final long UPDATE_NOTIFICATION_DELAY = 100;
 
-    private static String buildTitle(Context context, Messages.ListItem listItem) {
-        String departureTime = buildTickedDepartureTime(
-                context, listItem.getDepartureTime(), listItem.getSecondsLeft());
-        return departureTime + " " + listItem.getDepartureName();
+    private static String buildTitle(Messages.ListItem listItem) {
+        return listItem.getStopName() + " → " + listItem.getDepartureName();
+    }
+
+    private static int getSmallIcon(Messages.ListItem listItem){
+        int minLeft = listItem.getSecondsLeft()/60;
+        switch (minLeft){
+            case 1:
+                return R.drawable.icon_countdown_one;
+            case 2:
+                return R.drawable.icon_countdown_two;
+            case 3:
+                return R.drawable.icon_countdown_three;
+            case 4:
+                return R.drawable.icon_countdown_four;
+            case 5:
+                return R.drawable.icon_countdown_five;
+            case 6:
+                return R.drawable.icon_countdown_six;
+            case 7:
+                return R.drawable.icon_countdown_seven;
+            case 8:
+                return R.drawable.icon_countdown_eight;
+            case 9:
+                return R.drawable.icon_countdown_nine;
+            default:
+                if ((minLeft >= 10) && (minLeft < 15)) {
```

Comment:
start the if backwards, and half of the checks can be omitted
if (mnLeft > 45) {
} else if (minleft >= 30) {
  ...
}